themes/custom/droopler_theme/scss/components/_d_p_reference_content.scss

```diff
@@ -24,7 +24,10 @@ $d-p-reference-content-content-padding: 5px !default;
     padding-top: $d-p-reference-content-padding-top;
     padding-bottom: $d-p-reference-content-padding-bottom;
 
-    margin-right: -$d-p-reference-content-item-padding;
+
+    @include media-breakpoint-up(xl) {
+      margin-right: -$d-p-reference-content-item-padding;
+    }
```

Comment:
this one is because i noticed on homepage for mobile there is small bug in the blog references section - on the right side  tiles were touching screen